resources/js/svir_viewer_d3_tree.js

```diff
@@ -666,6 +676,62 @@
                     updateButton(pdId);
                 });
 
+            // Write (ignore weights) in a new line, if present
```

Comment:
perhaps use 'render' in place of 'write', first time I look at this it was not so clear